spec.bs

```diff
@@ -543,6 +556,260 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
 
 </div>
 
+<div algorithm="construct a pending fenced frame config">

```

Comment:
I don't think the algorithm value of `="...."` is necessary since the first thing in the algorithm is a definition.

Reply:
Sure, I'll remove it. There are a couple other algorithms in this file which I must have patterned off of...